pkg/option/option.go

```diff
@@ -133,6 +133,11 @@ type Option struct {
 	// PathOverrides optionally allows to override the default paths ghw uses internally
 	// to learn about the system resources.
 	PathOverrides PathOverrides
+
+	// Context may contain a pointer to a `Context` struct that is constructed
+	// during a call to the `context.WithContext` function. Only used internally.
+	// This is an interface to get around recursive package import issues.
+	Context interface{}
```

Comment:
While I totally see why you are doing this, this is a smell. There are options (no pun intended) to fix this, but all of them will require a overhaul of how we handle `context` and `option`, which must be either a blocker for 1.0.0 or deferred post 1.0.0 when we can break compatibility.

Reply:
agreed. I plan to merge option and context packages before 1.0

Reply:
Awesome. I'd love to help here.